backend/app/main.py

```
@@ -31,6 +42,32 @@ def create_app() -> FastAPI:
         redoc_url="/redoc",
     )
 
+    # 0. Global exception handlers
+    @app.exception_handler(Exception)
+    async def global_exception_handler(request: Request, exc: Exception):
```

Ruff (0.14.7)

Unused function argument: `request`

(ARG001)